INSTALL.txt

```diff
@@ -133,28 +119,31 @@
     Rakudo compiler is basically working and that it's capable of running a
     simple test harness.
 
-    Running "make spectest" will import the official Perl 6 test suite from
-    the "roast" repository <http://github.com/perl6/roast/> and run all
-    of these tests that are currently known to pass.
+    Running "make spectest" will import the official Raku test suite from the
+    "roast" repository <http://github.com/Raku/roast/> and run all of these
+    tests that are currently known to pass.
 
     At present we do not have any plans to directly store the official test
-    suite as part of the Rakudo repository, but will continue to fetch it
-    from the roast repository.  Releases of Rakudo get a snapshot of
-    the roast repository as of the time of the release.
+    suite as part of the Rakudo repository, but will continue to fetch it from
+    the roast repository.  Releases of Rakudo get a snapshot of the roast
```

Comment:
This sentence reads as if roast would be bundled with releases. AFAIK this is not the case.
What's probably meant is: "Releases of Rakudo will fetch a snapshot of the roast repository as of the time of the release."

Reply:
This was confusing me from the beginning. Actually, the whole paragraph needs to be changed.